src/Containers/Pages/Login/Login.js

```diff
@@ -1,9 +1,40 @@
-import React from "react";
+import React, {useState} from 'react';
+import './Login.css';
+import logo from "./googleicon.png";
+import firebase from '../../../firebase'
+  
+//   import { config } from "./test-credentials";
 
 class Login extends React.Component {
-  render() {
-    return <div>Login</div>;
-  }
+
+    // not used for now
+    state = {
+        signedIn: false,
+        user: null,
+    }
+
+    googleSignIn () {
+        const googleAuthProvider = new firebase.auth.GoogleAuthProvider();
+        firebase.auth().signInWithPopup(googleAuthProvider);
+        console.log("Google Sign In Page");
+    }
+
+    helpNeeded () {
+        console.log("Help is needed");
+    }
+
+    render () {
+        return(
+            <div className="loginComponent">
+                <h1 className="loginTitle">Login</h1>
+                <p className="loginParagraph">Welcome Back</p>
+                <p className="loginParagraph">Sign in to continue</p>
```

Comment:
Instead of using direct import of the CSS file, try importing it as a module. Later on, it becomes an issue when classes share the same name, then the styling also gets overwritten.  Look at here https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/